server/src/main/java/org/jboss/as/server/deployment/service/SecuredServiceRegistry.java

```diff
@@ -15,7 +16,10 @@
  */
 public class SecuredServiceRegistry implements ServiceRegistry {
 
-    public static final RuntimePermission PERMISSION = new RuntimePermission("org.jboss.as.server.deployment.service.USE_SERVICE_REGISTRY");
+    /**
+     * A {@link org.jboss.as.server.security.ServerPermission} needed to use {@link org.jboss.as.server.deployment.service.SecuredServiceRegistry}, i.e. invoke its methods. The name of the permission is "{@code useServiceRegistry}."
+     */
+    public static final ServerPermission USE_SERVICE_REGISTRY = new ServerPermission("useServiceRegistry");
```

Comment:
Please revert the name change. It's not necessary and this is public API. The type change is unavoidable, but it's likely anyone using this constant would only care about the "Permission"  type anyway.